src/App.js

```diff
@@ -14,11 +15,59 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    choosenItem: '',
+  };
+
+  chooseItem = (event) => {
+    this.setState({ choosenItem: event.target.innerText });
+  };
```

Comment:
От сюди насправді можна той item передати що ти мапиш і не доведеться текс з елементу діставати..
```suggestion
  chooseItem = (item) => {
    this.setState({ choosenItem: item  });
  };
```
Нижче відповідно передати його треба